channels/tsmf/client/gstreamer/tsmf_gstreamer.c

```diff
@@ -55,8 +59,33 @@ const char* get_type(TSMFGstreamerDecoder* mdecoder)
 
 	if (mdecoder->media_type == TSMF_MAJOR_TYPE_VIDEO)
 		return "VIDEO";
-	else
+	else if (mdecoder->media_type == TSMF_MAJOR_TYPE_AUDIO)
 		return "AUDIO";
+	else
```

Comment:
I think we should keep the else, UNKNOWN is a valid value and it is possible for a decoder object to have that value for the media_type parameter if it has not been initialized.